Jenkinsfile

```diff
@@ -267,33 +270,47 @@ pipeline {
             node('master') {
                 script {
                     deleteDir()
-                    def logServer = [
-                        addr: env.LOG_SERVER,
-                        user: env.LOG_SERVER_USER,
-                        folder: env.LOG_SERVER_FOLDER,
-                        rootDir: env.LOG_ROOT_DIR
-                    ]
-                    def destDir = decideLogsDestination(logServer, env.ZUUL_UUID)
+                    def relLogsDstDir = logsRelPathBasedOnTriggerSource(env.JOB_NAME,
+                        env.BUILD_NUMBER, env.ZUUL_UUID)
 
                     dir('to_publish') {
                         unstash 'processedTestReports'
                         def logFilename = 'log.txt.gz'
-                        obtainLogFile(env.JOB_NAME, env.BUILD_ID, logFilename)
-                        publishToLogServer(logServer, ".", destDir)
+                        createCompressedLogFile(env.JOB_NAME, env.BUILD_NUMBER, logFilename)
+
+                        def auth = sshAuthority(env.LOG_SERVER_USER, env.LOG_SERVER)
+                        def src = "."
+                        def dst = logsDirInFilesystem(env.LOG_ROOT_DIR, env.LOG_SERVER_FOLDER, relLogsDstDir)
+                        publishToLogServer(auth, src, dst)
```

Comment:
Since this is an only instance of call to `publishToLogServer` and we always call it to publish current directory contents to log server, maybe we could name it `publishCurrDirContentsToLogServer` (or something shorter) and drop its second parameter.

What do you think about this?

If we drop the second parameter, then this https://github.com/Juniper/contrail-windows-ci/pull/152/files#diff-b382b2f2a729b6abdeb3de77c8abe5d8R9 code branch is really dead.